CONTRIBUTING.md

```diff
@@ -28,7 +28,7 @@ Maintainers are the people who have commit access to the RestInCode repository a
 
 ### Triaging a new Issue
 
-The majority of new issues will be to add new individuals to RiC. Their subject will be a real name and/or a handle. 
+The majority of new issues will be to add new individuals to RiC. Their subject will be a real name and/or a handle. **An Issue to add a person should never be closed**. It will be used long-term to add new data and discuss the current memorial of this individual. Use labels to identify what work if any needs to be completed on the current Issue/person.
```

Comment:
The majority of new GitHub Issues will be used to add new individuals to Rest In Code (RiC). The Issue subject will be a real name and/or a handle. **An Issue to add a person should never be closed**. It will be used long-term to add new data and discuss the current memorial of this individual. Anyone is welcome to contribute, including sharing links related to the person, personal stories, or general thoughts on their friends who have passed. Use labels to identify what work if any needs to be completed on the current Issue/person.

Reply:
Added